code/modules/research/designs/AI_module_designs.dm

```diff
@@ -271,4 +271,13 @@
 	materials = list(/datum/material/glass = 1000, /datum/material/gold = 2000)
 	build_path = /obj/item/aiModule/core/full/reporter
 	category = list("AI Modules")
-	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
\ No newline at end of file
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/board/harmony_module
```

Comment:
You appear to have yoinked before you looked here.

Reply:
ah shit, thanks for pointing that out 👍